packages/repository/src/mongoDB/MongoDBConnection.js

```diff
@@ -13,7 +13,7 @@ const logger = require('@cedalo/logger').create({ name: 'MongoDBConnection' });
 
 const defaultOptions = {
 	host: process.env.MONGO_HOST || 'localhost',
-	port: parseInt(process.env.MONGO_PORT, 10) || 27017,
+	port: parseInt(process.env.MONGO_PORT, 10),
```

Comment:
Is it necessary to remove the default port 27017? Can we not set the port to 0 via the environment variables?

Reply:
This should work for both cases:
`port: !!process.env.MONGO_PORT ? parseInt(process.env.MONGO_PORT, 10) : 27017`
Defaults to 27017 if the env var is not set. Otherwise it tries parsing the given string.

Reply:
Thanks @anreinha